ekm/signer_key_manager_test.go

```diff
@@ -710,7 +710,7 @@ func TestSignRoot(t *testing.T) {
 			Message:   msg,
 		}
 
-		err = signed.GetSignature().VerifyByOperators(signed, networkconfig.TestNetwork.Domain, spectypes.QBFTSignatureType, []*spectypes.Operator{{OperatorID: spectypes.OperatorID(1), PubKey: pk.Serialize()}})
+		err = signed.GetSignature().VerifyByOperators(signed, networkconfig.TestNetwork.Domain, spectypes.QBFTSignatureType, []*spectypes.Operator{{OperatorID: spectypes.OperatorID(1), SharePubKey: pk.Serialize()}})
```

Comment:
should we fill operator keys in tests to be consistent with production?

Reply:
I thought about it as well but specifically for this function it wont use the operator keys so I don't see a reason